__tests__/__snapshots__/react.js.snap

```diff
@@ -2061,6 +2052,7 @@ const ListOfSomething = props => {
       {Array.isArray(props.from) &&
         props.from.map((item, index) => (
           <Something
+            index={index}
```

Comment:
I accidentally updated this snapshot and wasn't sure how to revert it :grimacing:

Reply:
hehe that's ok, this is on master now so it's fine like that :)